platform/rct/hal/usb_stm32/usb_conf.h

```diff
@@ -69,7 +69,15 @@
 /* IMR_MSK */
 /* mask defining which events has to be handled */
 /* by the device application software */
-#define IMR_MSK (CNTR_CTRM  | CNTR_WKUPM | CNTR_SUSPM | CNTR_ERRM  | CNTR_SOFM \
+
+/* JC: Original configuration
+
+  #define IMR_MSK (CNTR_CTRM  | CNTR_WKUPM | CNTR_SUSPM | CNTR_ERRM  | CNTR_SOFM \
+                | CNTR_ESOFM | CNTR_RESETM )
+*/
+
+/* JC: WKUP/SUSPEND Functionality disabled */
+#define IMR_MSK (CNTR_CTRM | CNTR_ERRM  | CNTR_SOFM \
```

Comment:
So by not handling the suspend/wkup event then we wont' go to sleep?  Sounds good to me.